### PR TITLE
Upstream merge april 30

### PR DESCRIPTION
**Pull request by scottmillner**

### CODE REVIEW

File: CODEOWNERS [hunk 1]
```diff
@@ -1,23 +1,27 @@
 *                           @gakonst
```
Comment (arminsabouri) [inline CODEOWNERS]: Can we remove this file.

Comment (scottmillner) [reply CODEOWNERS]: done!

File: crates/blockchain-tree/src/blockchain_tree.rs [hunk 2]
```diff
@@ -1421,6 +1423,7 @@ mod tests {
     }
 
     #[test]
+    #[ignore]
```
Comment (arminsabouri) [inline crates/blockchain-tree/src/blockchain_tree.rs:1426]: TODO to fix consecutive_reorgs() test
not required for this PR.
but im marking it to make a ticket

File: crates/botanix-lib/src/mint_validation.rs [hunk 3]
```diff
@@ -92,19 +92,22 @@ pub fn parse_pegin_reth_log_topic(
     log: &reth_primitives::Log,
     logs: &[Log],
 ) -> Result<PeginData, MintConsensusError> {
-    let revm_log = log.into();
+    let revm_log = <reth_primitives::Log as Clone>::clone(&(*log)).into();
```
Comment (arminsabouri) [inline crates/botanix-lib/src/mint_validation.rs:95]: Can these clones just be log.clone()

**Review by arminsabouri — APPROVED** [review]
Took a look at btc wallet
Mint validation
Evm crate
and changes to consensus crate

File: crates/botanix-lib/src/mint_validation.rs (hunk 3 above)
Comment (scottmillner) [reply crates/botanix-lib/src/mint_validation.rs:95]: ive taken note of this and will implement it in my fix test branch im doing next

**Comment by scottmillner** [comment]
some GH actions keep timing out
ive run them all locally and they pass so going to merge

**Pull request merged**